build/wd_test.bzl

```diff
@@ -87,32 +87,20 @@ exit /b !TEST_EXIT!
 SH_TEMPLATE = """#!/bin/sh
 set -e
 
-cleanup() {
-    if [ ! -z "$SIDECAR_PID" ]; then
-        kill $SIDECAR_PID 2>/dev/null || true
-    fi
-}
-
-trap cleanup EXIT
-
-# Start sidecar if specified
-if [ ! -z "$(SIDECAR)" ]; then
-    "$(SIDECAR)" & SIDECAR_PID=$!
-    # Wait until the process is ready
-    sleep 3
+# Run supervisor to start sidecar if specified
```

Comment:
Can you explain what this does as a comment? We're creating environment variables to be used by ...